test/services/mapping/goodsshipment/consignment/GoodsLocationBuilderSpec.scala

```diff
@@ -15,31 +15,31 @@
  */
 
 package services.mapping.goodsshipment.consignment
-import forms.declaration.{GoodsLocation, GoodsLocationSpec}
+import forms.declaration.GoodsLocation
+import forms.declaration.GoodsLocationTestData.correctGoodsLocationJSON
 import org.scalatest.{Matchers, WordSpec}
 import uk.gov.hmrc.http.cache.client.CacheMap
 
 class GoodsLocationBuilderSpec extends WordSpec with Matchers {
 
   "GoodsLocationBuilder" should {
-    "correctly map GoodsLocation instance for supplementary journey " when {
+    
+    "correctly map GoodsLocation instance for supplementary journey" when {
+     
       "all data is supplied" in {
+        
         implicit val cacheMap: CacheMap =
-          CacheMap(
-            "CacheID",
-            Map(
-              GoodsLocation.formId ->
-                GoodsLocationSpec.correctGoodsLocationJSON
-            )
-          )
+          CacheMap("CacheID", Map(GoodsLocation.formId -> correctGoodsLocationJSON))
+        
         val goodsLocation = GoodsLocationBuilder.build
-        goodsLocation.getID.getValue should be("9GB1234567ABCDEF")
```

Comment:
Again we agreed as a team we are happy with the should be() approach and this just introduces inconsistency in our tests. Please leave as should be() unless agreed with team we are all moving to using shouldBe